src/jobs/test-nut.yml

```diff
@@ -31,7 +40,20 @@ steps:
   - install-sfdx:
       version: <<parameters.sfdx_version>>
       os: <<parameters.os>>
-  - checkout
+  - when:
+      condition:
+        equal: ['', <<parameters.repo_tag>>]
```

Comment:
what's the tag stuff used for?

Reply:
@mshanemc The repo_tag will be used to checked out the plugin source at a GH tag. The value will be the version that the plugin was in when released with sfdx-cli (or as close as possible.)

See the changes made below that adds the `git clone` when tag is not an empty string.